README.md

```diff
@@ -119,7 +119,13 @@ line.
 
 ## Installing from ISO
 
-[Download a Fedora CoreOS ISO image](https://getfedora.org/coreos/download/).
+Download a Fedora CoreOS ISO image:
+
+```
+podman run --privileged --pull=always --rm -v .:/data -w /data \
```

Comment:
Nit: is `--privileged` really needed for the `download` verb?

Reply:
Assuming `.` is under `$HOME`, SELinux will deny the write without it.